src/content/app/tools/blast/components/species-selector/SpeciesSelector.scss

```diff
@@ -0,0 +1,123 @@
+@import 'src/styles/common';
+
+.header {
+  display: flex;
+  width: 860px;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 28px;
+  padding-bottom: 12px;
+}
+
+.headerGroup {
+  display: flex;
+  align-items: center;
+}
+
+.headerTitle {
+  font-size: 15px;
+  font-weight: $bold;
+}
+
+.maxSpecies {
+  font-weight: $light;
+  margin-right: 30px;
+}
+
+.clearAll {
+  margin-right: 30px;
+  color: $blue;
+  cursor: pointer;
+  user-select: none;
+}
+
+.speciesCounter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 28px;
+  height: 28px;
+  color: $white;
+  background-color: $dark-grey;
+  font-weight: $bold;
+  border-radius: 100%;
+  font-size: 15px;
+  margin: 0 0.6rem 0 0.6rem;
+}
+
+.speciesSelectorPlaceholder {
+  width: 860px;
+  height: 684px;
```

Comment:
Does Andrea explicitly want this much of empty grey space under the last species? Is she ok with the box being shorter?

Reply:
This was in the dev spec. i will double check with her.

Reply:
Box height to grow as the page grows with some padding at the bottom.

Reply:
Done